packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -1022,6 +1017,15 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         );
         // Serialize the gas estimates as an ABI encoded `uint256` array.
         vm.serializeBytes(deploymentInfoKey, "gasEstimates", abi.encode(_deployment.gasEstimates));
+        // Serialize the Sphinx library version as an ABI encoded string. We ABI encode it to ensure
+        // that Foundry doesn't serialize it as a number, which will happen if the
+        // `sphinxLibraryVersion` consists only of numbers. If Foundry serializes it as a number,
+        // it'll be prone to the same precision loss due to JavaScript's low integer size limit.
+        vm.serializeBytes(
+            deploymentInfoKey,
+            "sphinxLibraryVersion",
+            abi.encode(_deployment.sphinxLibraryVersion)
+        );
```

Comment:
Since we use semver versions (i.e `v1.0.0`), isn't this unnecessary? 

I don't think doing this is a big deal to be clear, I'm just curious about the extent of the bug we're working around.

Reply:
It is unnecessary, but I intentionally did it this way out of an abundance of caution (e.g. maybe in the future we'll change it to a string that can contain all numbers)